core/src/main/java/jenkins/SmokeTest.java

```diff
@@ -0,0 +1,7 @@
+package jenkins;
+
+/**
+ * Interface used as a Category for JUnit tests
+ */
+public interface SmokeTest {
```

Comment:
Since you modify only `test/`, it should be a part of https://github.com/jenkinsci/jenkins-test-harness IMHO

Reply:
I moved it to lib-test-annotations and created a pull request. https://github.com/jenkinsci/lib-test-annotations/pull/1